docs/web-console-guide_5/master.adoc

```diff
@@ -57,10 +57,70 @@ include::topics_5/web-analyze-apps.adoc[leveloffset=+1]
 include::topics_5/web-add-project.adoc[leveloffset=+2]
 
 // Add Applications to a Project
-include::topics_5/web-add-applications.adoc[leveloffset=+2]
+// include::topics_5/web-add-applications.adoc[leveloffset=+2]
+
+=== Adding applications to a project
+
+A project must contain one or more applications.
+
+You can add applications to a project by:
+
+* Uploading applications
+* Registering the directory path of each application
+
+Uploading applications stores them directly in the {ProductShortName} server.
+
+Registering the directory path of an application tells the {ProductShortName} server the location of the application. This method is useful if you expect to update the application because it ensures that the {ProductShortName} server always uses the latest version of the application.
+
+.Adding applications
+image::web-add-apps_51.png[Adding applications]
+
+include::topics_5/web-uploading-apps.adoc[leveloffset=+3]
+include::topics_5/web-directory-path-apps.adoc[leveloffset=+3]
 
 // Specify the Analysis Configuration
-include::topics_5/web-execution-config.adoc[leveloffset=+2]
+
+=== Configuring an analysis
+
+You can configure an analysis by:
+
+* Selecting the transformation targets of the applications
+* Selecting Java application packages to be analyzed
+* Using custom rules or labels
+* Setting advanced options
++
+The first item is required. The rest are optional.
+The configuration interface includes a review page.
+
```

Comment:
Honestly, I don't think you need any of this intro above. Just tell the user that they can configure an analysis. I don't see how the rest adds anything.